### DISCUSSION TITLE
记录有道词典笔S6成功获取adb权限

### DISCUSSION THREAD

**Discussion by SkySight-666** [PenUniverse/PenMods#249] [奇技淫巧]
edited: 方法暂不公布，等待有道回应。

**Comment by Redbeanw44602** [comment]
edited: 方法暂不公布，等待有道回应。

**Reply by SkySight-666** [reply]
edited: 方法暂不公布，等待有道回应。

**Reply by Redbeanw44602** [reply]
edited: 方法暂不公布，等待有道回应。

**Reply by Redbeanw44602** [reply]
建议去反馈一下吧，另外这种东西修复前最好不要公开披露，对你不利。（帮你edit掉了）
https://aq.163.com/submit

**Reply by SkySight-666** [reply]
> 建议去反馈一下吧，另外这种东西修复前最好不要公开披露，对你不利。（帮你edit掉了） https://aq.163.com/submit

OK，谢谢大佬

**Reply by SkySight-666** [reply]
> 建议去反馈一下吧，另外这种东西修复前最好不要公开披露，对你不利。（帮你edit掉了） https://aq.163.com/submit

![image](https://github.com/user-attachments/assets/6dae4593-ad6b-4d1d-b9b4-843cda2dc165)
![image](https://github.com/user-attachments/assets/3a653662-24dc-4eb6-8fe7-05e48bfad9ca)

**Reply by SkySight-666** [reply]
> 建议去反馈一下吧，另外这种东西修复前最好不要公开披露，对你不利。（帮你edit掉了） https://aq.163.com/submit

我的OS变4.0.3了 不知道是我自己不小心调的还是wy推送了一个不通知的强制更新，打开系统更新会卡好一会儿，然后发现4.0.3（原来是4.0.2）

**Reply by Redbeanw44602** [reply]
哈哈，离谱。
这样，漏洞类型选 客户端漏洞---其他，危害等级选高危，然后在下面说明 **有道硬件** 以及 **中间人攻击**以及操作系统会**自动更新**，再不管就算了。

**Reply by SkySight-666** [reply]
> 哈哈，离谱。 这样，漏洞类型选 客户端漏洞---其他，危害等级选高危，然后在下面说明 **有道硬件** 以及 **中间人攻击**以及操作系统会**自动更新**，再不管就算了。

好的，前几天在考试 没弄（其实我第一次也写的和这个差不多，就是离奇）

**Reply by SkySight-666** [reply]
> 哈哈，离谱。 这样，漏洞类型选 客户端漏洞---其他，危害等级选高危，然后在下面说明 **有道硬件** 以及 **中间人攻击**以及操作系统会**自动更新**，再不管就算了。

![image](https://github.com/user-attachments/assets/c3ff3b6e-a742-4d20-9648-4447c8a7e8b4)
搞笑

**Reply by Redbeanw44602** [reply]
你可以新开一个帖子放出方法了既然他们不认为这是漏洞，我将lock此主题。
